tutorials/dataframe/df106_HiggsToFourLeptons.py

```diff
@@ -60,40 +55,45 @@
 }
 """)
 
-for s in samples:
-    # Select electron or muon trigger
-    df[s] = df[s].Filter("trigE || trigM")
+# Select electron or muon trigger
+df = df.Filter("trigE || trigM")
 
     # Select events with exactly four good leptons conserving charge and lepton numbers
     # Note that all collections are RVecs and good_lep is the mask for the good leptons.
     # The lepton types are PDG numbers and set to 11 or 13 for an electron or muon
     # irrespective of the charge.
-    df[s] = df[s].Define("good_lep", "abs(lep_eta) < 2.5 && lep_pt > 5000 && lep_ptcone30 / lep_pt < 0.3 && lep_etcone20 / lep_pt < 0.3")\
-                 .Filter("Sum(good_lep) == 4")\
-                 .Filter("Sum(lep_charge[good_lep]) == 0")\
-                 .Define("goodlep_sumtypes", "Sum(lep_type[good_lep])")\
-                 .Filter("goodlep_sumtypes == 44 || goodlep_sumtypes == 52 || goodlep_sumtypes == 48")
 
-    # Apply additional cuts depending on lepton flavour
-    df[s] = df[s].Filter("GoodElectronsAndMuons(lep_type[good_lep], lep_pt[good_lep], lep_eta[good_lep], lep_phi[good_lep], lep_E[good_lep], lep_trackd0pvunbiased[good_lep], lep_tracksigd0pvunbiased[good_lep], lep_z0[good_lep])")
+df = df.Define("good_lep", "abs(lep_eta) < 2.5 && lep_pt > 5000 && lep_ptcone30 / lep_pt < 0.3 && lep_etcone20 / lep_pt < 0.3")\
+            .Filter("Sum(good_lep) == 4")\
+             .Filter("Sum(lep_charge[good_lep]) == 0")\
+             .Define("goodlep_sumtypes", "Sum(lep_type[good_lep])")\
+             .Filter("goodlep_sumtypes == 44 || goodlep_sumtypes == 52 || goodlep_sumtypes == 48")
```

Comment:
The indentation here looks a bit weird. Can it be so that every new line is aligned with the `.Define` of the first line?